### ISSUE TITLE
do libtorch support torch.nn.functional.interpolate?

### ISSUE THREAD

**Issue by zhangyingbit** [pytorch/pytorch#44188]
I have trained a instance segmentation model **Yolact** in python (linux) , then I want to deploy it to c++ (windows platform).
First I convert the .pth to .pt file. Then in the c++, I met some errors , so I spend some time to figure out what is wrong.

The backbone(resnet101) and fpn work fine in c++.
But in the mask prototypes branch, there is a upsampling step
 ```
layer = InterpolateModule(scale_factor=-kernel_size, mode='bilinear', align_corners=False, **layer_cfg[2])
```
which InterpolateModule is defined by
```
class InterpolateModule(nn.Module):
	def __init__(self, *args, **kwdargs):
		super().__init__()
		self.args = args
		self.kwdargs = kwdargs
	def forward(self, x):
		return F.interpolate(x, *self.args, **self.kwdargs)
```
In c++ ,  there is a "std::runtime_error" because the **InterpolateModule**  operation. Can anyboby tell me that whether libtorch support this operation? How can I modify **InterpolateModule** to make it work?

my windows configure is below:
 - PyTorch Version (e.g., 1.0): 1.1.0
 - OS (e.g., Linux): windows 10
 - How you installed PyTorch (`conda`, `pip`, source): in fact I download **libtorch 1.1.0**
 - Build command you used (if compiling from source): cmake
 - Python version: 3.6.5
 - CUDA/cuDNN version: 9.0/7.5
 - GPU models and configuration: RTX 2070

Thank you very much!!!

**Comment by zhangyingbit** [comment]
I find libtorch 1.1.0 doesn't support the interpolate operator,  libtorch 1.4.0 works for me, hope it can help you: )
